libsrc/apple2/cpuspeed.s

```diff
@@ -0,0 +1,40 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2024
+;
+; void __fastcall__ apple2_slowdown(void)
+; void __fastcall__ apple2_speedup(void)
```

Comment:
Are you aware of https://github.com/cc65/cc65/blob/master/include/accelerator.h?
You may be surprised to find quite some references to Apple II in https://github.com/cc65/cc65/blob/master/include/accelerator.h#L45-L55

Some background: Someone needed to implement tests for the accelerator support in VICE. And there's was a momentum to implement those tests as cc65 library code in order to have a synergy effect. It was planned to extend the cc65 library to support accelerators on other targets as well. However, for "some reason or another" this didn't happen. And what too didn't happen, was to implement some umbrella interface that allows the user to interact with the accelerator speed independently from the actual accelerator.

So my proposal is to implement three functions
`set_iigs_speed()`
`get_iigs_speed()`
`detect_iigs()`
and add them to https://github.com/cc65/cc65/blob/master/include/accelerator.h

Reply:
I was not! I'll investigate that and rework the PR.
Thanks !